eslint.config.mjs

```diff
@@ -0,0 +1,235 @@
+import _import from 'eslint-plugin-import'
+import { fileURLToPath } from 'node:url'
+import { fixupPluginRules } from '@eslint/compat'
+import { FlatCompat } from '@eslint/eslintrc'
+import eslint from '@eslint/js'
+import globals from 'globals'
+import jest from 'eslint-plugin-jest'
+import json from '@eslint/json'
+import markdown from '@eslint/markdown'
+import mdx from 'eslint-plugin-mdx'
+import path from 'node:path'
+import react from 'eslint-plugin-react'
+import tseslint from 'typescript-eslint'
+import tsParser from '@typescript-eslint/parser'
+import tsPlugin from '@typescript-eslint/eslint-plugin'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: eslint.configs.recommended,
+  allConfig: eslint.configs.all,
+})
+
+export default tseslint.config(
+  // Global
+  {
+    ignores: [
+      '**/node_modules/',
```

Comment:
I don't think this is necessary, this is part of [the default patterns](https://eslint.org/docs/latest/use/configure/ignore#ignoring-files)